lib/ts/string_view.h

```diff
@@ -1210,3 +1211,19 @@ operator<<(std::basic_ostream<_Type, _Traits> &os, const basic_string_view<_Type
 using string_view = basic_string_view<char>;
 
 } // namespace ts
+
+// suffix for basic_string_view literals
+constexpr ts::string_view operator"" _sv(const char *str, size_t len) noexcept
```

Comment:
I am concerned this will not work on some of our compilers. This (as in sv literal is added in C++17) was not added by myself because I thought one of the systems we support will not handle it. This should be in a namespace as well

Reply:
While the `_sv` literal in specific was added in C++17, the user literal mechanism is in C++11 AFAICT. It does seem to compile correctly on all of the PR level compilers, which is a good variety. I'm not sure why this should be in a namespace - what is the benefit of that? The literal tag seems to be global.

Reply:
I tried moving the string literal definition in to the `ts` namespace and it doesn't get found when used. Therefore we should leave it at global scope.